### DISCUSSION TITLE
Feature request: Clock panel with date

### DISCUSSION THREAD

**Discussion by ollie-campbell** [grafana/grafana#60568] [Plugins]
I've just started using the clock panel, which is really useful, but it would be great if the date and timezone could be on the same line as the time. At the moment they sit above and below.

Ideally, they could be sat above and below, until the height of the clock panel is reduced (by the user when it's applied to a dashboard) and they then slide onto the same line.

**Comment by tolzhabayev** [comment]
Hey @ollie-campbell,
please check out the version [2.1.2]( https://github.com/grafana/clock-panel/releases/tag/v2.1.2) - https://grafana.com/grafana/plugins/grafana-clock-panel/ 

It might be doing exactly what you are requesting.

**Reply by ollie-campbell** [reply]
Perfect, thank you.
